fireblocks_sdk/tokenization_api_types.py

```diff
@@ -49,9 +49,15 @@ def __init__(self, inputs: List[ParameterWithValue], name: Optional[str] = ""):
 
 
 class EVMTokenCreateParams(BaseDictClass):
-    def __init__(self, contract_id: str, constructor_params: Optional[List[ParameterWithValue]] = None):
+    def __init__(
+            self, 
+            contract_id: str, 
+            constructor_params: Optional[List[ParameterWithValue]] = None, 
```

Comment:
if you already remove the `get_contract_template_constructor` endpoint, why not removing also the param?